pkg/runner/runner.go

```diff
@@ -74,6 +77,31 @@ func Run(cfg *Config) error {
 		tput.InitWatcher(cfg.ThroughputInterval)
 	}
 
+	if cfg.TracerCfg.JaegerCollectorEndpoint != "" {
```

Comment:
We should make it easy to configure with either Jaeger OR open telemetry endpoint

Reply:
Adding new types of endpoints is left for next iteration, Jaeger was needed because that what we use in our current setup.

Reply:
If we put this off, let's add an issue to keep track of this.

Reply:
https://github.com/timescale/promscale/issues/1076